api/http/routing.go

```diff
@@ -102,9 +105,15 @@ func NewRouter(ctx context.Context, c config.Reader,
 
 	app := app.NewDeployments(mongoStorage, fileStorage, app.ArtifactContentType)
 
-	apiConf := NewConfig()
-	if key, err := base64.StdEncoding.DecodeString(
-		c.GetString(dconfig.SettingPresignSecretBase64),
+	// Encode base64 secret in either std or URL encoding ignoring padding.
+	base64Repl := strings.NewReplacer("-", "+", "_", "/", "=", "")
```

Comment:
Maybe it is late, but I do not understand this replacing, why is it so?

Reply:
It converts `base64.URLEncoding` to `base64.RawStdEncoding`. This way we can decode all 4 variants of base64 encoding (with or without padding).